maker-frontend/src/MakerApp.tsx

```diff
@@ -124,6 +125,22 @@ export default function App() {
                             </HStack>
                         </HStack>
 
+                        <Text>Long Price:</Text>
+                        <HStack>
+                            <CurrencyInputField
+                                onChange={(valueString: string) => {
+                                    setLongPrice(parse(valueString));
+                                }}
+                                value={format(longPrice)}
+                            />
+                            <HStack>
+                                <Switch
+                                    disabled={true}
+                                />
+                                <Text>Auto-refresh</Text>
+                            </HStack>
```

Comment:
you might as well set it up and fill the autorefresh with the `bid` price , adjusting for the spread.
(or remove the auto-refresh stuff altogether here, if you're not planning to add it).